eos-tech-support/eos-write-live

```diff
@@ -0,0 +1,272 @@
+#!/bin/bash -e
+set -o pipefail
+
+# Writes an Endless OS image to a device (e.g., USB drive or SD card) so that it
+# can be used both as a live system, and to install itself. The image is
+# extracted to an exFAT partition, and is booted using a GRUB and
+# initramfs which know how to loopback-mount the image.
+#
+# NOTE: the partition table will not be in physical order so Windows can mount
+# the image partition. Additional images copied to the image partition will be
+# detected by the installer, but will not be bootable.
+
+EOS_WRITE_IMAGE=$(dirname $0)/eos-write-image
+if [ ! -f $EOS_WRITE_IMAGE ]; then
+    EOS_WRITE_IMAGE='eos-write-image'
+fi
+EOS_DOWNLOAD_IMAGE=$(dirname $0)/eos-download-image
+if [ ! -f $EOS_DOWNLOAD_IMAGE ]; then
+    EOS_DOWNLOAD_IMAGE='eos-download-image'
+fi
+
+ARGS=$(getopt -o o:x:,lnmwfh -l "os-image:,windows-tool:,latest,ntfs,mbr,writable,force,help" -n "$0" -- "$@")
```

Comment:
Apparently it makes `-,` an acceptable option. I'll fix this and the other instance of it in `eos-write-installer`.